packages/devextreme/js/__internal/ui/html_editor/ui/aiDialog.ts

```diff
@@ -113,7 +137,10 @@ export default class AiDialog extends BaseDialog<AiDialogResult> {
         this._commandOptionsList = this._commandsMap[e.value]?.options ?? [];
         this._currentOption = this._commandOptionsList?.[0];
 
-        this._syncDialogWithState();
+        this._isAskAICommandSelected = e.value === 'askAI';
```

Comment:
✨ **[minor: magic string]**

'askAI': Let's use constants to store string literals.